resources/views/bread/edit-add.blade.php

```diff
@@ -63,11 +63,11 @@
                                 <div class="form-group">
```

Comment:
Field views should start at `<div class="form-group">`. Actually the loop is inside a single `.form-group`, which is not right, it should be outside and each item should be wrapped by that class.